src/domain/Tables/Table/Table.tsx

```diff
@@ -134,35 +89,42 @@ const Table = <T extends{}>(props: ITableProps<T>) => {
   } = props
 
   validateProps(props)
-  const [selectedAll, setSelectedAll] = useState<TableCheckboxInputValue>(TableCheckboxInputValue.False)
```

Comment:
`selectedAll` was an uneccessary state here (I think it was needed for other features because those remembered select all between pages, while this doesn't care)